docs/sphinx/about/index.txt

```diff
@@ -59,9 +59,42 @@ than being updated whenever a new version of :omerodoc:`OMERO <>` is released.
 We expect this to result in more frequent releases to get fixes out to the
 community faster.
 
-The version number is three numbers separated by dots e.g. 4.0.0. See the
-:doc:`version history <whats-new>` for a list of major changes in each
-release.
+See the :doc:`version history <whats-new>` for a list of major changes in
+each release.
+
+.. _versioning-policy:
+
+Versioning policy
+^^^^^^^^^^^^^^^^^
+
+Bio-Formats does not currently conform to `strict semantic versioning <http://semver.org>`_.
+The following set of rules describe the current policy (using
+`RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_):
+
+- The version number MUST take the form X.Y.Z where X, Y, and Z are
+  non-negative integers, and MUST NOT contain leading zeroes.
+- The patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards
```

Comment:
I copied this from Seb's design text and can remove it if it's confusing. An x.y.0 release isn't a patch release though, is it.